app/controllers/ansible_credential_controller.rb

```diff
@@ -21,7 +21,7 @@ def self.model
   end
 
   def display_repositories
-    nested_list(ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptSource)
+    nested_list(ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource)
```

Comment:
These changes were necessary so we don't collide with `Workflow::AutomationManager::ConfigurationScriptSource`